website/docs/admin_server_licenses.md

```diff
@@ -0,0 +1,48 @@
+---
+id: admin_server_licenses
+title: AYON Licenses
+sidebar_label: AYON Licenses
+description: Managing licenses and why you might need them
+---
+
+The AYON server operates under a [Functional Source License (FSL)](https://fair.io/), ensuring that most features remain free for users.
+
+Some AYON features and addons require a license, which can be obtained through an [AYON subscription](https://ynput.io/ayon/pricing/) or granted on a per-instance basis. For example, PowerPack features require an AYON subscription.
+
+An AYON subscription includes a set number of seats, limiting how many users can log in and use AYON. This limit is called a **license pool**, and you can have multiple pools of different types.
+
+## Fixed User Pools
+
+The primary pool type is a **fixed user pool**, where a set number of seats are assigned to specific users.
+
+For example, a yearly [AYON Pro subscription](https://ynput.io/ayon/pricing/) for 20 users provides a fixed pool of 20 seats, allowing 20 assigned users to log in.
```

Comment:
We need to mention here, that they can change which users are occupying the seats from any pool the license at will.